src/Main.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react'
+import ProfilesPage from './ProfilesPage.js'
+import LandingPage from './LandingPage.js'
+import LogInPage from './LogInPage.js'
+import SignUp from './SignUp.js'
+import firebase from './firebase.js'
+
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+
+import * as actions from './actions.js'
+
+const LANDING = 'landing'
+const LOG_IN = 'login'
```

Comment:
You also defined these in `reducers.js` ... not a good practice to define them in both places. What if you accidentally did `const LOG_IN = 'logn'` ? You would have some seriously difficult bugs to figure out. Better to export these constants from `reducers.js` and import them here.

Reply:
fixed